libqtile/scripts/cmd_obj.py

```diff
@@ -192,11 +192,12 @@ def add_subcommand(subparsers, parents):
         """\
     Examples:
      qtile cmd-obj
-     qtile cmd-obj -o cmd
-     qtile cmd-obj -o cmd -f prev_layout -i
-     qtile cmd-obj -o cmd -f prev_layout -a 3 # prev_layout on group 3
+     qtile cmd-obj -o root
+     qtile cmd-obj -o root -f prev_layout -a 3 # prev_layout on group 3
      qtile cmd-obj -o group 3 -f focus_back
-     qtile cmd-obj -o cmd -f restart # restart qtile
+     qtile cmd-obj -o root -f restart # restart qtile
+    The graph traversal recurses:
+     qtile cmd-obj -o screen 0 -o bar bottom -f root -o screen -o group -o window -o root -f status
```

Comment:
This line look wrong to me. To traverse the graph I'd just do one `-o` argument e.g. `qtile cmd-obj -o screen 0 bar bottom`.

There's no way to return to the root node from any other object so, if you actually get an output of `"OK"` here it's because it's running `status` on just the `-o root` part and it's ignoring everything else.